deployment/experiment-deployment/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3.7'
+
+services:
+  nginx:
+      build:
+        context: .
+        dockerfile: /gtc-nginx/Dockerfile
```

Comment:
Das funktioniert so wahrscheinlich nicht, oder?

Reply:
Ich würde das Image in GitHub Actions bauen und dann auf den Experiment-Server kopieren, vgl. https://stackoverflow.com/questions/23935141/how-to-copy-docker-images-from-one-host-to-another-without-using-a-repository

Dann kannst du das Image mit seinem Namen aufrufen

Reply:
Siehe oben, das kann man viel einfacher lösen. Vergiss, was ich hier gesagt habe :-)